docs/src/main/sphinx/connector/hive.rst

```diff
@@ -705,6 +705,20 @@ connector.
         also have more overhead and increase load on the system.
       - ``64 MB``
 
+.. _hive-table-redirection:
+
+Table redirection
+-----------------
+
+.. include:: table-redirection.fragment
+
+When dealing with a non Hive table, the connector supports
```

Comment:
The connector supports redirection from Hive tables to Iceberg and Delta Lake tables with the following catalog configuration properties: